dashboard/src/components/EnvComponents/Features.tsx

```diff
@@ -8,86 +8,90 @@ const Features = ({ variables, setVariables }: any) => {
 		<div>
 			{' '}
 			<Text fontSize="md" paddingTop="2%" fontWeight="bold" mb={5}>
-				Disable Features
+				Features
 			</Text>
 			<Stack spacing={6}>
 				<Flex>
 					<Flex w="100%" justifyContent="start" alignItems="center">
-						<Text fontSize="sm">Disable Login Page:</Text>
+						<Text fontSize="sm">Login Page:</Text>
 					</Flex>
 					<Flex justifyContent="start">
 						<InputField
 							variables={variables}
 							setVariables={setVariables}
 							inputType={SwitchInputType.DISABLE_LOGIN_PAGE}
+							is_Disable={true}
```

Comment:
1. We should use camel-case (`isDisabled`) or snake-case (`is_disabled`). To maintain the consist I would recommend using camel-case.
2. is_Disabled name makes it feel as if the input is disabled as it is prop for input field. Instead, we should rename it to `hasReversedValue`.
3. You don't need to pass `true` value to boolean props by default if you just add `hasReversedValue` it will consider it true